### ISSUE TITLE
windows pip install  UnicodeDecodeError: 'gbk' codec can't decode byte

### ISSUE THREAD

**Issue by skydazzle** [aliyun/aliyun-log-python-sdk#59]
long_description = open('README.md').read()
UnicodeDecodeError: 'gbk' codec can't decode byte 0xad in position 847: illegal multibyte sequence

**Comment by wjo1212** [comment]
thanks for reporting this issue. it's fixed and checked into master branch, will be released in next release. 

please verify it via:
```shell
pip install -U git+https://github.com/aliyun/aliyun-log-python-sdk.git@master
```

**Comment by wjo1212** [comment]
fixed in 0.6.21
